lib/print.mjs

```diff
@@ -12,62 +12,112 @@ function deleteUserDataDir(dirPath) {
   });
 }
 
+let browser = null;
+let page = null;
+
+let methods = [];
+let options = {};
+let url = '';
```

Comment:
These variables shouldn't be defined in the module context as their values will be overriden everytime `print` is called.

It is possible to call `print` method again before the previous `print` method has completed its execution. This can then cause the previous `launchPage` and `generatePdf` to be called with incorrect `url`, `options` and `methods`,

#### Illustration

1. `Request 1` - `print`, `url` = A
1. `Request 1` - `launchBrowser`
1. `Request 2` - `print` `url` = B
1. `Request 3` - `print` `url` = C
1. `Request 1` - `launchPage` - will be called with `url` = C
1. `Request 1` - `generatePdf`